core/Controller/AppPasswordController.php

```diff
@@ -42,26 +42,14 @@
 use OCP\Security\ISecureRandom;
 
 class AppPasswordController extends \OCP\AppFramework\OCSController {
-	private ISession $session;
-	private ISecureRandom $random;
-	private IProvider $tokenProvider;
-	private IStore $credentialStore;
-	private IEventDispatcher $eventDispatcher;
-
 	public function __construct(string $appName,
 								IRequest $request,
```

Comment:
```suggestion
	public function __construct(
		string $appName,
		IRequest $request,
```
If we go with this refactor I’d prefer if we use this indentation method to avoid crazy empty spaces.
Also the last one can have a comma as well, resulting in clearer diff when we add a property later.

Reply:
I think I might have messed up something in this PR regarding the spaces. Sorry about that.

Although I like the idea of always having a last comma, I just tried to make them consistent with the existing conventions in the controller classes. No personal preference intended. I appreciate your attention to this matter.